package.json

```diff
@@ -17,6 +17,10 @@
     "@babel/core": "7.11.6",
     "@babel/plugin-transform-runtime": "7.11.5",
     "@babel/preset-env": "7.11.5",
+    "@graph-paper/core": "0.0.0-alpha.15",
+    "@graph-paper/datagraphic": "0.0.0-alpha.15",
+    "@graph-paper/elements": "0.0.0-alpha.15",
+    "@graph-paper/guides": "0.0.0-alpha.15",
```

Comment:
These are the packages I ended up pulling in for my example project so I think having these in the app template would be perfect.